src/plugins/presentation_util/public/components/solution_toolbar/items/popover.tsx

```diff
@@ -44,14 +41,21 @@ export const SolutionToolbarPopover = ({
     />
   );
 
+  children.props = {
+    ...children.props,
+    onClick: () => {
+      closePopover();
+    },
+  };
+
```

Comment:
Had to add this to force the menu popover to close when one of the options is selected.

Note that, before this, the `SolutionToolbarPopover` was **only** used for listing all of the visualization types - however, since the `onClick` of these options often leads to a different app, the popover remaining open didn't really matter. The **only** instance where this bug was present before this change was when adding `Log stream` visualizations - since this doesn't navigate to another app, you'll notice that the popover stays open. This fixes this for both `Log stream` visualizations **and** the new controls dropdown.